### PR TITLE
[feature-157803976] Implement add Cohort to a society

### PR DESCRIPTION
**Pull request by georgreen**
- Add put method to endpoint `/api/v1/societies/cohorts`

## Resolves #<issue number>
- #157803976
## Description (what problem you're fixing)

  - Linking users to their respective societies
  

## Fix (what you did to fix it)

  - Implement a CLI tool to link Cohorts to their respective societies
  - Implement an endpoint to allow admins to add cohorts to socities

## How to test (describe how to test your PR)

  - `$ python manage.py tests`

### CODE REVIEW

File: src/api/utils/marshmallow_schemas.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
     Schema, fields, post_load, validates,
     validate, ValidationError, validates_schema
 )
-from api.models import User, Activity, ActivityType, Role
+from api.models import User, Activity, ActivityType, Role, Cohort, Society
```
[the remark this answers is not in the document]
Comment (Andretalik) [reply src/api/utils/marshmallow_schemas.py]: I think this should be resolved.

File: src/api/utils/helpers.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def find_item(data):
     ), 404)
 
 
-def response_builder(data, status_code):
+def response_builder(data, status_code=200):
```
Comment (Andretalik) [inline src/api/utils/helpers.py]: I think the status code should be explicitly stated by the developers who is building the response.

File: src/manage.py [hunk 3]
```diff
@@ -68,21 +81,42 @@ def seed():
...
-manager.add_command("shell", Shell(make_context=shell))
-manager.add_command("db", MigrateCommand)
+@manager.command
+def link_society_cohort(cohort_name, society_name):
+    """CLI tool, link cohort with society."""
+    with app.app_context():
+        cohort = Cohort.query.filter_by(name=cohort_name).first()
+        if not cohort:
+            return print(
+                f'Error cohort by name: {cohort_name} does not exit in DB.')
```
Comment (Andretalik) [inline src/manage.py]: 'Error cohort by name: {cohort_name} does not exit in DB.' should be 'Error cohort by name: {cohort_name} does not exist in DB.'

**Comment by georgreen** [comment]
@maquchizi  Coday flags methods wrongly, I need my verbs(GET, PUT...etc) to be instance methods.

**Comment by Andretalik** [comment]
@georgreen But where in that function are class attributes used with in it?
I think that's why Codacy is complaining. When it is a @classmethod none of these issues will be apparent.
It still works though.

**Comment by georgreen** [comment]
@Andretalik Yes, it complaining because self has not been referenced anywhere in the method, changing it to a class method makes the error on Coday to go away but does not fix it in any way. I will not use the cls parameter either. Using the verbs as methods on the resources is a cleaner way to model our endpoints.

File: src/api/utils/helpers.py (hunk 2 above)
Comment (georgreen) [reply src/api/utils/helpers.py]: Yes but most of the status_code will be 200, I don't think everyone should repeat that every time.

**Pull request merged**